vendor/bat-native-ads/src/bat/ads/internal/ml/data/vector_data_aliases.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ML_DATA_VECTOR_DATA_ALIASES_H_
+#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ML_DATA_VECTOR_DATA_ALIASES_H_
+
+#include <utility>
+
+namespace ads {
+namespace ml {
+namespace data {
+
+using SparseVectorElement = std::pair<unsigned, double>;
```

Comment:
see https://google.github.io/styleguide/cppguide.html#Integer_Types in regards to use of unsigned

Reply:
Done, used uint32_t